include/pages/login.inc.php

```diff
@@ -1,20 +1,25 @@
 <?php
 $defflip = (!cfip()) ? exit(header('HTTP/1.1 401 Unauthorized')) : 1;
 
+$recaptcha_enabled = ($setting->getValue('recaptcha_enabled') && $setting->getValue('recaptcha_enabled_logins'));
```

Comment:
Good idea, never had this idea when writing MPOS.

Reply:
The perks of being lazy ;)